src/dropdown/dropdown.js

```diff
@@ -10,6 +10,7 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
   this.open = function( dropdownScope ) {
     if ( !openScope ) {
       $document.bind('click', closeDropdown);
+      $document.bind('touchstart', closeDropdown);
```

Comment:
You can instead add touchstart to the above binding like: `$document.bind('click touchstart', closeDropdown);`